shark_turbine/kernel/ops/wave_ops.py

```diff
@@ -386,6 +390,13 @@ def index(self, value: Any):
         else:
             raise ValueError("Index must be a dict")
 
+    def post_expansion(self, constraints: list["Constraint"]) -> None:
```

Comment:
This PR used to have pre-expansion mechanisms right? Just curious, are we saving it for later? haha

Reply:
NVM, I asked for this before, please ignore!